content/blog/pragmatism-over-perfection.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Pragmatism Over Perfection"
+authorId: "aleric"
+date: 2025-04-21
+draft: false
+featured: true
+weight: 1
+---
+
+We as Engineers often chase perfection.
+
+It fuels our curiosity, sharpens our skills, and makes us feel good about the things we build. But at the same time, it can be a double-edged sword, sometimes slowing us down or distracting us from what actually matters: **delivering business value**.
```

Comment:
While your sentences are to the point and short I suggest creating small paragraphs instead of having individual sentences by themselves. I feel it would provide a natural flow while reading.